gossip/emitter/emitter.go

```diff
@@ -62,7 +62,6 @@ type Emitter struct {
 	payloadIndexer *ancestor.PayloadIndexer
 
 	intervals                EmitIntervals
-	globalConfirmingInterval time.Duration
```

Comment:
Seem like a mis-implementation because of the config struct contains
```go
// EmitIntervals is the configuration of emit intervals.
type EmitIntervals struct {
	Min                        time.Duration
	Max                        time.Duration
	Confirming                 time.Duration // emit time when there's no txs to originate, but at least 1 tx to confirm
	ParallelInstanceProtection time.Duration
	DoublesignProtection       time.Duration
}
```
I believe this interval need to be used somewhere

Reply:
It can be easily re-added